### DISCUSSION TITLE
add new feature - avatar group

### DISCUSSION THREAD

**Discussion by ikesela** [fomantic/Fomantic-UI#2045] [Elements (button, image, label, loader, ...)]
CSS example: 
`.avatar-group .avatar {
  position: relative;
  z-index: 2;
  border: 2px solid #fff; }
  .avatar-group .avatar:hover {
    z-index: 3; }

.avatar-group .avatar + .avatar {
  margin-left: -1rem; }`

Code Example:

 `<div class="avatar-group">
     <img src="https://via.placeholder.com/150" class="ui mini avatar image"> 
     <img src="https://via.placeholder.com/150" class="ui mini avatar image"> 
     <img src="https://via.placeholder.com/150" class="ui mini avatar image"> 
     <img src="https://via.placeholder.com/150" class="ui mini avatar image"> 
     <img src="https://via.placeholder.com/150" class="ui mini avatar image"> 
  </div> `

**Comment by lubber-de** [comment]
🤫 This is in progress....Wait for 2.9.0 😉
